templates/blog/product-cards.html

```diff
@@ -26,6 +26,16 @@ <h3>
   <p>Kubernetes, or K8s for short, is an open source platform pioneered by Google, which started as a simple container orchestration tool but has grown into a platform for deploying, monitoring and managing apps and services across clouds.</p>
   <p><a href="/kubernetes/what-is-kubernetes">Learn more about Kubernetes &rsaquo;</a></p>
 </div>
+{% elif article.tags and 3680 in article.tags %}
+<div class="p-card" id="rtp-bosch">
+<img alt="IoT infographic" src="https://assets.ubuntu.com/v1/53163a88-IOT_core_infographic.svg" class="">
+<hr class="u-sv1">
+  <h3>
+    <a href="/internet-of-things">Ubuntu for the Internet of Things</a>
+  </h3>
+  <p>From smart homes to smart drones, robots, and industrial systems, Ubuntu is the new standard for embedded Linux. Get the world’s best security, a custom app store, a huge developer community and reliable updates.</p>
+  <p><a href="/internet-of-things">Learn more ›</a></p>
```

Comment:
Can you encode the > with `&nbsp;&rsaquo;` please

Reply:
Oh indeed, lazy copy paste.